setup.py

```diff
@@ -25,7 +25,9 @@
     author="shobrook",
     author_email="shobrookj@gmail.com",
     # classifiers=[],
-    install_requires=[],
+    install_requires=[
+        "json_repair",
```

Comment:
I think `"json-repair"` would be correct here. Also, could you please add `"openai"` and `"tiktoken"` here too?